socorro/processor/rules/breakpad.py

```diff
@@ -141,11 +138,13 @@ def __init__(
         symbols_urls=None,
         command_path="/stackwalk-rust/minidump-stackwalk",
         command_line=(
-            "timeout --signal KILL {kill_timeout} {command_path} "
-            "--raw-json {raw_crash_path} "
+            "timeout --signal KILL {kill_timeout} "
+            "{command_path} "
+            "--raw-json={raw_crash_path} "
+            "--symbols-cache={symbol_cache_path} "
+            "--symbols-tmp={symbol_tmp_path} "
             "{symbols_urls} "
-            "--symbols-cache {symbol_cache_path} "
-            "--symbols-tmp {symbol_tmp_path} "
+            "--verbose=error "
```

Comment:
It's unenthusing that this is in both places, but the configuration is in a different file and I like having the defaults here because it's at hand.

At some point, I'll figure out a better way to do this.